scripts/create_addon

```diff
@@ -56,7 +56,7 @@ pack_addon() {
     fi
     cd $ADDON_BUILD
     echo "*** compressing Addon $PKG_ADDON_ID ... ***"
-    zip -rq $PKG_ADDON_ID-$ADDONVER.zip $PKG_ADDON_ID;
+    $TOOLCHAIN/bin/7za a -l -mx9 -bsp0 -bso0 $PKG_ADDON_ID-$ADDONVER.zip $PKG_ADDON_ID;
```

Comment:
Any special reason for ; character at the end of line?
Also why doesn't we store symbolic links as links? Is kodi even capable to extract them correctly? I think this could save space in some cases.